### DISCUSSION TITLE
Radio listings

### DISCUSSION THREAD

**Discussion by BellezaEmporium** [iptv-org/community#83] [Show and tell]
On the IPTV pace, I've decided to create a listing for radios : https://github.com/LaneSh4d0w/IPRD .

It'll be heavily inspired from this repository for the different ways we're doing stuff (m3u files, way we identify streams), and it's in a **VERY EARLY WIP** state, just to see if everything works correctly. 

Of course, it won't grow at the same speed than what this repository has become, but I was interested on giving a single and unique place to get all your favorite radios.

And of course, if the organisation wants, i'll attach this repo to the organisation.

**Comment by sguinetti** [comment]
Interesting. The only thing allowed in this repository, from what I'm seeing, are visual radios. Combining the review of the daily bots helps quite a bit. Still there are already websites that use PD-license database like https://www.radio-browser.info/.

**Reply by BellezaEmporium** [reply]
> Interesting. The only thing allowed in this repository, from what I'm seeing, are visual radios. Combining the review of the daily bots helps quite a bit. Still there are already websites that use PD-license database like https://www.radio-browser.info/.

Radio-Browser is basically what I wanted to achieve, but in a portable format.

Yes, since this repository is axed in television, mine is off-topic basically. But I wanted to shoot my shot and try to put as much streams as I could. I also know a Discord bot that has got quite a good amount of radios, and I suspect it using Radio-Browser's developer API for getting them.

The community (and channel growth) has been quite strong thanks to multiple channel additions that gained interest for multiple people. (I do also remember that there were a few channel owners updating their channels in here for the sake of other viewers)

**Comment by BellezaEmporium** [comment]
After some time, it's finally out ! The links are there, now I'm focusing on splitting links for each service :)
